src/Adapters/Laravel/Repositories/DatabaseAnalyticsRepository.php

```diff
@@ -31,15 +31,13 @@ public function __construct(private PanConfiguration $config)
     public function all(): array
     {
         /** @var array<int, Analytic> $all */
-        $all = DB::table('pan_analytics')->get()->map(fn (mixed $analytic): Analytic => new Analytic(
+        return DB::table('pan_analytics')->get()->map(fn (mixed $analytic): Analytic => new Analytic(
```

Comment:
Hello, 

I'm not sure you have run `composer test` before submiting your PR. I've got 2 errors. One with Rector and one with PHPStan

```
1) src/Adapters/Laravel/Repositories/DatabaseAnalyticsRepository.php:29

    ---------- begin diff ----------
@@ @@
      */
     public function all(): array
     {
-        /** @var array<int, Analytic> $all */
         return DB::table('pan_analytics')->get()->map(fn (mixed $analytic): Analytic => new Analytic(
             id: (int) $analytic->id, // @phpstan-ignore-line
             name: $analytic->name, // @phpstan-ignore-line
    ----------- end diff -----------

Applied rules:
 * RemoveNonExistingVarAnnotationRector


 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   Adapters/Laravel/Repositories/DatabaseAnalyticsRepository.php                                                                                       
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------- 
  33     Method Pan\Adapters\Laravel\Repositories\DatabaseAnalyticsRepository::all() should return array<int, Pan\ValueObjects\Analytic> but returns array.  
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------- 
```